cohorts/load.py

```diff
@@ -1020,6 +1020,93 @@ def plot_joint(self, on, on_two=None, **kwargs):
         p = sb.jointplot(data=df, x=plot_cols[0], y=plot_cols[1])
         return p
 
+    def summarize_provenance(self):
```

Comment:
What do you think about calling this `cache_to_provenance` and the other one (gathered across caches) `summarize_provenance`; the `unique` vs. non-unique didn't really help me understand what was going on. Open to alternatives!